src/string_bytes.cc

```diff
@@ -493,15 +508,29 @@ Maybe<size_t> StringBytes::Size(Isolate* isolate,
     case UCS2:
       return Just(str->Length() * sizeof(uint16_t));
 
-    case BASE64URL: {
-      String::Value value(isolate, str);
-      return Just(simdutf::base64_length_from_binary(value.length(),
-                                                     simdutf::base64_url));
-    }
-
+    case BASE64URL:
     case BASE64: {
-      String::Value value(isolate, str);
-      return Just(simdutf::base64_length_from_binary(value.length()));
+      size_t data_size = str->Length() % 4 <= 1
```

Comment:
Given the duplication here, any chance of moving this into a separate helper function?